samcli/lib/test_runner/invoke_testsuite.py

```diff
@@ -0,0 +1,116 @@
+"""
+Kicks off a testsuite in a Fargate container
```

Comment:
Are there going to be other types of `SuiteRunner`s? You may want to consider making an interface and extend it to be Fargate. This gives you the ability to make different `SuiteRunner`s and interchange them with a factory or something similar.

An example of this is our "provider" for getting different types of resources parsed out of templates: https://github.com/aws/aws-sam-cli/blob/develop/samcli/lib/providers/provider.py#L472.

I wouldn't require this now but something you may want to consider having in place, which could make this much more expandable in the future.

Reply:
Great idea, will do